pandas/core/arrays/categorical.py

```diff
@@ -917,7 +916,8 @@ def reorder_categories(self, new_categories, ordered=None, inplace=False):
 
         Returns
         -------
-        cat : Categorical with reordered categories or None if inplace.
+        cat : Categprocal or None
```

Comment:
Typo here. Otherwise, lgtm

Reply:
Thank you. Done.
 I start working on other docstring fixes in the same way.

Reply:
Note that CI Checks complains about missing dots at the end of return statement.
```
6818:RT05:pandas.DataFrame.interpolate:Return value description should finish with "."
```

Reply:
I'm sorry for all the mistakes. Thank you for pointing that out.